src/main/java/com/minelittlepony/hdskins/gui/GuiSkinsMineLP.java

```diff
@@ -22,26 +26,26 @@ protected EntityPlayerModel getModel(GameProfile profile) {
     }
 
     @Override
-    protected void onSetLocalSkin(MinecraftProfileTexture.Type type) {
+    protected void onSetLocalSkin(Type type) {
         MineLittlePony.logger.debug("Invalidating old local skin, checking updated local skin");
-        if (type == MinecraftProfileTexture.Type.SKIN) {
-            ponyManager.removePony(this.localPlayer.getSkinTexture());
+        if (type == Type.SKIN) {
+            ponyManager.removePony(localPlayer.getSkinTexture());
         }
     }
 
     @Override
-    protected void onSetRemoteSkin(MinecraftProfileTexture.Type type, ResourceLocation location, MinecraftProfileTexture profileTexture) {
+    protected void onSetRemoteSkin(Type type, ResourceLocation resource, MinecraftProfileTexture profileTexture) {
```

Comment:
Unneeded name change

Reply:
*needed name change

I'd rather the source code be consistent with its naming schema.